gfsphysics/GFS_layer/GFS_radiation_driver.F90

```diff
@@ -2058,12 +2059,18 @@ subroutine GFS_radiation_driver                             &
       if (Model%lssav) then
         if (Model%lsswr) then
           do i=1,im
-            Diag%fluxr(i,34) = Diag%fluxr(i,34) + Model%fhswr*aerodp(i,1)  ! total aod at 550nm
-            Diag%fluxr(i,35) = Diag%fluxr(i,35) + Model%fhswr*aerodp(i,2)  ! DU aod at 550nm
-            Diag%fluxr(i,36) = Diag%fluxr(i,36) + Model%fhswr*aerodp(i,3)  ! BC aod at 550nm
-            Diag%fluxr(i,37) = Diag%fluxr(i,37) + Model%fhswr*aerodp(i,4)  ! OC aod at 550nm
-            Diag%fluxr(i,38) = Diag%fluxr(i,38) + Model%fhswr*aerodp(i,5)  ! SU aod at 550nm
-            Diag%fluxr(i,39) = Diag%fluxr(i,39) + Model%fhswr*aerodp(i,6)  ! SS aod at 550nm
+!            Diag%fluxr(i,34) = Diag%fluxr(i,34) + Model%fhswr*aerodp(i,1)  ! total aod at 550nm
```

Comment:
I am not seeing this change in lines 2062 to 2073 in CCPP - is this not required?

If it is, what do you want to do with it? Also fix as part of our NCAR dtc/develop to EMC develop PR?

Reply:
Made this change in NCAR's dtc/develop branch (ccpp-physics). Will come to NCAR master with the next commit from our side.